docs/pages/tutorial/follow-up.mdx

```diff
@@ -39,7 +40,9 @@ Once we've built a project we're ready to share with users, we can build it into
 
 ### Navigation
 
-We had one screen in the tutorial app. Most apps have multiple screens. See [Expo Router](/router/introduction/) for more information about adding navigation to your app.
+We had one screen in the tutorial app. Most apps have multiple screens. [Expo Router](/router/introduction/) allows you to add new screens to your app by adding additional files to your project. Get started with Expo Router already configured by creating your next app with the default template:
```

Comment:
```suggestion
We had one screen in the tutorial app. Most apps have multiple screens. [Expo Router](/router/introduction/) allows you to add new screens to your app by adding additional files to your project. To create a new project with Expo Router library already installed and configured, see [Create a project](/get-started/create-a-project/).
```

Let's add a link to the Get Started section instead of specifying the command here. Since the tutorial targets new users, this is a good point for them to go explore Setting their environment (Expo Go vs Development builds) in the next guide after create a project.